libs/common/src/test-helpers/mock-data/utilisation-report.entity.mock-builder.ts

```diff
@@ -66,7 +66,7 @@ export class UtilisationReportEntityMockBuilder {
     return this;
   }
 
-  public withAzureFileInfo(azureFileInfo: AzureFileInfoEntity): UtilisationReportEntityMockBuilder {
+  public withAzureFileInfo(azureFileInfo: AzureFileInfoEntity | undefined): UtilisationReportEntityMockBuilder {
```

Comment:
is this for like 'marked as completed' reports? wondering when we would want to call a test with no azure file info aka why it accepts undefined - probably more for my own understanding

Reply:
This is specifically for "marked as completed" reports (replied because I've added this in my PR as well)

Reply:
Yeah that's right, the mock builder by default adds azure file info if the status you give it is anything other than REPORT_NOT_RECEIVED. So if you ever want to test anything without azure file info and a status other than REPORT_NOT_RECEIVED you need to explicitly set it to undefined.